src/content/changelog/4-13-1-2020-02-10.md

```diff
@@ -0,0 +1,38 @@
+---
+title: '4.14.0: 2020-02-10'
+date: 2020-02-10T05:00:00.000Z
+version: 4.14.0
+changelog:
+  - API
+---
+### Added
+
+- Added LKE node event `lke_node_create` to Event action list. See action information in the Event endpoints: 
+  - `List Events` ([GET /account/events](https://developers.linode.com/api/v4/account-events))
+  - `View Event` ([GET /account/events/{eventId}](https://developers.linode.com/api/v4/account-events-event-id)) 
+
+### Changed
```

Comment:
Looks like the linode->node language change in the LKE pool response is missing.

Reply:
In just the List Node Pools endpoint? this one is "broken" :( sadly it doesn't ever show this array. It has been changed to show "nodes" instead of "linodes".